pepper-apis/docs/specification/src/components/schemas.yml

```diff
@@ -1713,6 +1788,53 @@ Question.Picklist:
         type: array
         items:
           $ref: '#/Answer.Picklist'
+Question.Matrix:
+  allOf:
+    - $ref: '#/Question'
+    - type: object
+      required:
+        - selectMode
+        - matrixOptions
+        - matrixQuestions
+        - answers
+      properties:
+        questionType:
+          type: string
+          default: MATRIX
+          enum:
+            - MATRIX
+        selectMode:
+          type: string
+          enum:
+            - MULTIPLE
+            - SINGLE
+        groups:
+
+          type: array
+          description: A list of groups used in the `matrixOptions` values. If there are no options which explicity request grouping, this will be an empty array.
+          minItems: 0
+          items:
+            type: object
+            required:
+              - identifier
+              - name
+            properties:
+              identifier:
+                type: string
+              name:
+                type: string
+        matrixOptions:
+          type: array
+          items:
+            $ref: '#/MatrixOption'
+        matrixQuestions:
+          type: array
+          items:
+            $ref: '#/MatrixRowQuestion'
```

Comment:
What is this used for? I don't see any references to a question identifier in other parts of the question, and the `MatrixRowQuestion` doesn't refer to any `groups` or `matrixOptions`

Reply:
In Matrix question, `matrixQuestions` basically are rows, when `matrixOptions` are columns.